srun: error: ioctl(TIOCGWINSZ): Inappropriate ioctl for device
srun: error: Not using a pseudo-terminal, disregarding --pty option
[ERROR] File already exists: /raid/local/containers/enroot-data/user-1787/pytorch
Traceback (most recent call last):
  File "/home/projects/fraudagent/hase-michael/V3/test.py", line 34, in <module>
    main()
  File "/home/projects/fraudagent/hase-michael/V3/test.py", line 12, in main
    test_df = pd.read_csv(test_csv)
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/parsers/readers.py", line 1026, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/parsers/readers.py", line 620, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/parsers/readers.py", line 1620, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/parsers/readers.py", line 1880, in _make_engine
    self.handles = get_handle(
  File "/usr/local/lib/python3.10/dist-packages/pandas/io/common.py", line 873, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'test_set.csv'
srun: error: klb-dgx-002: task 0: Exited with exit code 1
